docs/src/documentation/05-development/04-migration-guides/v9.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: V9
+description: How to migrate to Orbit 9.0.0
+redirect_from:
+  - /migration-guides/v9/
+---
+
+# Orbit Migration Guide v9
+
+This migration guide focuses on the process of migrating from Orbit v<LATEST 8 VERSION> to v9.0, as some breaking changes were introduced.
```

Comment:
`v<LATEST 8 VERSION>` is this a placeholder to change before merge/release?

Reply:
Before release, yes. Though we could remove it in case it causes more trouble for us :) (or update it after the facts)